wd.sh

```diff
@@ -255,7 +255,10 @@ wd_remove()
 
 wd_browse() {
     if ! command -v fzf >/dev/null; then
-        echo "This functionality requires fzf. Please install fzf first."
+        echo "This functionality of wd requires fzf. Please install fzf" \
+             "first or disable the use of fzf setting" \
+             "`bindkey ${FZF_WD_BINDKEY:-'^B'} backward-char` in your" \
```

Comment:
Don't backticks have some special meaning in shell scripts, like execute what's in them and interpolate the result into the string or something? Similar to `$()`

Reply:
Yes, my goal is to make this quoted to say "copy paste this exactly into your zshrc". Basically, the zshrc should be:
```sh
# Some other code
bindkey ${FZF_WD_BINDKEY:-'^B'} backward-char

# more other code
```

The reason I didn't use single quotes is that there's a single quote around `^B`. So it seems like `\"bindkey... \"` would be best and then add "(without the quotes)". What do you think?

Reply:
Yeah I'd be happy with that. Or if you'd prefer to overhaul this PR to do something more like what's being discussed in #133 I'd be up for that too. Your choice!